README.md

```diff
@@ -145,14 +145,13 @@ MacOs            | Chrome
 Zonemaster GUI is tested manually and with testing tools. See the
 [Zonemaster-gui repository][Zonemaster-GUI] for more details.
 
-## Support of DNSSEC algorithms 15 and 16
+## Support of DNSKEY algorithms 15 and 16
```

Comment:
Table of content needs to be updated as well

Reply:
I missed that. Thanks.